docs/russian/github-for-noobs.md

```diff
@@ -1,173 +0,0 @@
-## Инструкция по работе с сайтом/приложением "GitHub":

```

Comment:
Why did you remove the russian documentation? Many people found this very useful. Also this has nothing to do with gulp.